shared/src/test/diff/nu/MethodSignatures.mls

```diff
@@ -184,10 +184,11 @@ fun f(x: Int): Int
 fun f(Int): Int
 //│ fun f: Int -> Int
 
+// FIXME
```

Comment:
This is actually expected. Why the fixme?

Reply:
So is there a `:e` missing?

Reply:
Ah yes, good catch.